src/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java

```diff
@@ -210,6 +214,33 @@ protected void EnableForeignKeyConstraints(Connection conn) {
     }
   }
 
+  protected void CreatePgSqlProcedures(Connection conn) throws SQLException {
```

Comment:
nit: maybe just say `CreateSqlProcedures` since this isn't Postgres (rather Yugabyte/YSQL).
Ditto about using `PGSQL` in the summary.

Reply:
Done.